configure

```diff
@@ -2313,11 +2295,7 @@ echo $_no_undefined_var_template
 # macros.
 # Note: as above, we check for -Wpragma-pack as not all compilers error out on
 # unknown -Wno-xxx flags.
-echocheck "whether C++ compiler accepts -Wno-pragma-pack"
-echo "int main() { return 0; }" >$TMPC
-cc_check -Wpragma-pack -Werror && _no_pragma_pack=yes
-test "$_no_pragma_pack" = yes && append_var CXXFLAGS "-Wno-pragma-pack"
-echo $_no_pragma_pack
+set_flag_if_supported -Wno-pragma-pack
```

Comment:
This is not the same as what have been done before.
Before, we checked support of the positive flag and enabled the negative one.

Reply:
Does it matter?

Reply:
Ok, I see it does :)

I'll try to find a better way.